packages/accordions/src/styled/timeline/StyledSeparator.ts

```diff
@@ -10,43 +10,23 @@ import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden
 
 const COMPONENT_ID = 'timeline.content.separator';
 
-interface IStyledSeparator {
-  surfaceColor?: string;
-}
-
-/**
- * 1. override CSS bedrock
- */
 export const StyledSeparator = styled.div.attrs({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION
-})<IStyledSeparator>`
+})`
   display: flex;
   position: relative;
   justify-content: center;
   padding: ${props => `${props.theme.space.base * 5}px ${props.theme.space.base}px`};
 
-  svg {
-    box-sizing: content-box; /* [1] */
-  }
-
-  svg,
-  img {
-    position: relative;
-    z-index: 2;
-    background: ${props => props.surfaceColor || props.theme.colors.background};
-    padding: ${props => props.theme.space.base}px 0;
-    color: ${props => getColor('neutralHue', 600, props.theme)};
-  }
```

Comment:
☝️ When possible, always prefer to add CSS directly against the styled component for which it applies, rather than relying on descendent selectors which are a bit more brittle to DOM structural changes.